src/dbnode/client/session.go

```diff
@@ -255,7 +257,18 @@ func newSession(opts Options) (clientSession, error) {
 		},
 		metrics: newSessionMetrics(scope),
 	}
-	s.reattemptStreamBlocksFromPeersFn = s.streamBlocksReattemptFromPeers
+	s.reattemptStreamBlocksFromPeersFn = func(
+		reattemptBlocks []receivedBlockMetadata,
+		reEnqueueCh enqueueChannel,
+		err error,
+		reattemptReason reason,
+		reattemptType reattemptType,
+		m *streamFromPeersMetrics,
+	) error {
+		return s.streamBlocksReattemptFromPeers(
+			reattemptBlocks, reEnqueueCh, err,
+			reattemptReason, reattemptType, m)
+	}
```

Comment:
nit: This can just be `s.reattemptStreamBlocksFromPeersFn = s.streamBlocksReattemptFromPeers` yeah?

Reply:
lol yeah, that was kind of a mindless refactor. Fixed.